src/components/Button/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './index.css'
+
+
+const btnStyle = {
+  backgroundColor: '#FFEBD9',
+  color: "black"
+};
+
+function Button(props) {
+  return (
+    <div>
```

Comment:
You can use `<React.Fragment>` or its shorthand `<>` and `</>` to replace the `div` here unless you have specific styles you want to apply to this tag.